vertical-pod-autoscaler/examples/hamster.yaml

```diff
@@ -3,14 +3,15 @@
 # 0.5s and sleeping 0.5s).
 # It also creates a corresponding Vertical Pod Autoscaler that adjusts the requests.
 # Note that the update mode is left unset, so it defaults to "Auto" mode.
-apiVersion: "autoscaling.k8s.io/v1beta1"
+apiVersion: "autoscaling.k8s.io/v1beta2"
 kind: VerticalPodAutoscaler
 metadata:
-  name: hamster-vpa
+  name: hamster-vpa-beta2
```

Comment:
I think it would be slightly better if we kept the naming here as was and add -deprecated suffix to all objects in the other yaml.
This way a) it would be harder to miss deprecation b) there would be no conflicts between the two examples.
WDYT?

Reply:
+1

Reply:
Good point. Done